ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ip-routing.j2

```diff
@@ -1,24 +1,22 @@
 {# eos - IP Routing #}
-{% if ip_routing is defined and ip_routing is not none %}
-{%     if ip_routing == true %}
+{% if ip_routing is arista.avd.defined %}
+{%     if ip_routing is arista.avd.defined(true) %}
 !
 ip routing
 {%     else %}
 !
 no ip routing
 {%     endif %}
 {% endif %}
-{% if vrfs is defined and vrfs is not none %}
+{% if vrfs is arista.avd.defined %}
 {%     for vrf in vrfs | arista.avd.natural_sort %}
-{%             if vrfs[vrf].ip_routing == true and vrf is ne 'default' %}
+{%             if vrfs[vrf].ip_routing is arista.avd.defined(true) and vrf is ne 'default' %}
```

Comment:
Too many spaces for indentation here: 8 instead of 4

Reply:
fix indentation